WooCommerce/Classes/Tools/PaymentsPluginsInfoProvider.swift

```diff
@@ -0,0 +1,69 @@
+import Yosemite
+import Foundation
+import Storage
+
+struct PaymentsPluginsInfoProvider {
```

Comment:
I like that you've moved this logic out.

Did you consider putting it in the Yosemite layer? I don't think there's a problem with having it in the app layer (since that's where it came from) but it seems like it would fit better in Yosemite. That can be done later to avoid pushing the timescales out, if you have any concerns about that: this is a hot fix after all.

I'm still learning my way around the architecture so another opinion would be welcome! @ctarda do you have any thoughts on this?

Reply:
I am a bit confused about this struct. I do not see it in the project file in this diff, and I do not see it in the Xcode project tree if I check out the branch. Is this actually being used?

Reply:
It got renamed, sorry, I was going commit by commit when I reviewed. From the next commit it's called `PaymentsPluginsDataProvider`

Reply:
Exactly, as @joshheald mentioned, I renamed to `PaymentsPluginsDataProvider`, a more meaningful name IMHO